### ISSUE TITLE
[a11y audit] Memorials Self-Service - VA Form 40-1330M (Desktop and Mobile)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#87533]
> [!CAUTION]
> Launch-blocking issues encountered during audit.

# CAIA Accessibility Audit
CAIA will review the  MSS VA Form 40-1330M (DocuSign) in both mobile and desktop environments, for accessibility. We will make note of any issues encountered and provide recommendations to the MSS Team.

## Audit(s) Needed
The [VA Form 40-1330M](https://demo.services.docusign.net/webforms-ux/v1.0/forms/4e3fda343cdaa54263be7213ae0026ee) needs audited on both mobile and desktop.
- [x] **Mobile** - completed by @SarahKay8 
- [x] **Desktop** - completed by @coforma-jamie 

### Audit Docs

[Google Doc: Audit Spreadsheet](https://docs.google.com/spreadsheets/d/1elZ23024sbp6Zi0Kpazir7jQRtAqn5NTDgeMY34Q0RM/edit?gid=1370873395#gid=1370873395) - _Only go as far as the “Summary” screen, we don’t need to audit the actual signature process (yet)._
- [x] Mobile
- [x] Desktop

## About the MSS Team

<details><summary>Toggle learn more about the team</summary>

- Team name: Memorials Self-Service
- OCTO product owner: Matt Self
- Product name: Apply for a medallion in a private cemetery
- Product manager:  Catherine Hughes
- Slack channel: [#sitewide-content-accessibility-ia](https://join.slack.com/share/enQtNzI1MjcxNzM4MDIxMi0zOWVhNjIyMWVhNDZlMzkzNmMzY2E2MzM1ZDMzYjViYmQ3YzU4OTE3ZTZhMjkzNDI4OTc2MzM3NTJhMWE0NWY2)
- Dedicated content writer on your team (if you have one): N/A, UX Team
- Dedicated a11y specialist on your team (if you have one): N/A, UX Team
</details>

## About VA Form 40-1330M
 
<details><summary>Toggle to read original intake ticket description</summary>

> We hope to have accessibility and content specialist analyze the work we've conducted so far regarding VA Form 40-1330M, Apply for a medallion in a private cemetery form. In this form, we create a new digital form applying for a medallion to fix to a private headstone or marker.
> 
> We hope to have the content team analyze the content decisions we've made so far, and provide recommendations for further improvements. This includes the entire medallions form, and the two potential solutions we have so far for multiple signatures. You can add comments in the Figma file as needed.
> 
> We hope to have accessibility team members assess the entire medallions form, and provide accessibility recommendations for further improvements. We ask the accessibility team to look over this DocuSign link to test for accessibility for ongoing feedback, [DocuSign link](https://demo.services.docusign.net/webforms-ux/v1.0/forms/4e3fda343cdaa54263be7213ae0026ee). If the accessibility team could prioritize this DocuSign effort, it would be much appreciated.
> 
> Note: There will be another DocuSign link to add to the ticket to assess this week. Update: The link has been added and updated.

</details>

## Definition of Done

```[tasklist]
### A11Y Audit Tasks
- [x] [CAIA Internal Slack Thread Jamie/SK](https://oddball.slack.com/archives/C05LB36ATAQ/p1719947272142779)
- [x] [Create Audit Spreadsheet](https://docs.google.com/spreadsheets/d/1elZ23024sbp6Zi0Kpazir7jQRtAqn5NTDgeMY34Q0RM/edit?gid=1370873395#gid=1370873395) - Jamie
- [x] Desktop Audit - Jamie
- [x] Mobile Audit - SK
- [x] Secondary audit of sign and submit (see [spreadsheet](https://docs.google.com/spreadsheets/d/1nV6iehJ3kq2FcQkhcC-1yyuwJa1bYM2laQvZjPpcMJg/edit?gid=0#gid=0))
```

```[tasklist]
### Deliverable
- [x] Completed accessibility review with feedback/recommendations 🔗 (link goes here to markdown file [example](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Veteran%20Facing%20Forms/2024%2007%2002%20Audit-%20VFF%20-%20VA%20Form%2021-4138.md) or put in comments of this ticket.
- [x] Notify team in comments
- [ ] Notify team in Slack
- [ ] Add to agenda for Kickoff 7/10 check with Lily or Naomi (no agenda as of 7/2/2024)
```

**Comment by sara-amanda** [comment]
Updated ticket with links to agenda and deliverables.

**Comment by sara-amanda** [comment]
# High Priority Items from A11Y Audit 

The following are the critical and/or high priority items found in the audits of the form and sign and submit. 
- **Form**: 67% of issues identified are critical or major. 
- **Sign and Submit:** 100% of issues identified are critical or major. 

## Findings Overview 

### Form Expected Experience Standards

| Anticipated Severity | Count |
|--------|--------|
Critical (1, LB) | 1 
Major (2) | 10
Major (3) (SUBJECT to Docusign) | 1
Minor (4) | 3
Moderate (3) | 3
Grand Total | 18

### Submit and Sign Expected Experience Standards

| Anticipated Severity | Count |
|--------|--------|
Critical (1, LB) | 1 
Major (2) | 5
Grand Total | 6

## Expected Experience Standards 

### Form

<details><summary>Toggle to view the expected experience standards for the form</summary>

- User can complete necessary flows
- User can determine next steps in a flow
- User can perceive all meaningful elements
- User can review, edit or confirm their information before form submission
- User encounters no repetition or redundancy
- User experiences predictable interactions with components
- User has enough information to complete a task
- User knows where a button or link leads
</details>

### Sign and Submit

<details><summary>Toggle to view the expected experience standards for sign and submit</summary>

- User can perceive all meaningful elements
- User experiences predictable interactions with components

</details>

## Issue Descriptions

### Form

<details><summary>Toggle to view the issue descriptions for the form</summary>

- Critical (1):  
    - Focus isn't managed - screen changes aren't announced
- Major (2): 
    - Applicant Information: Hint text not contained within label
    - Introduction: Non-descriptive alt text
    - Mobile Audit - Environment - Talkback, Android + Chrome Incorrect Screenreader Output ("Phone No." )
    - Mobile Audit Environment- Talkback Android + Chrome - Incorrect Screen Reader Output for Form Title 
    - Mobile Audit Environment- Talkback Android + Chrome-  Race Options - Screenreader Output Issue 
    - Mobile Audit Environment- Talkback Android + Chrome- Redundant Screen Reader Output for Text Box Instructions
    - No separate URL / route for each screen of the form
    - Summary: "Edit" buttons don't actually allow you to edit
    - Text size is defined using pixels; don't change when user changes browser settings
    - Use of disabled
- Major (3): 
    - Mobile Audit Environment - Talkback Android + Chrome DOCUSIGN Issue (Just adding for visibility) - Issues Escaping Menu Pop up 

</details>

### Sign and Submit

<details><summary>Toggle to view the issue descriptions for sign and submit</summary>

- Critical (1):  
    - Zooming and scaling disabled
- Major (2): 
    - Before clicking "I agree" and "continue": Hidden, unusable H3
    - Mobile Audit - Android Talkback - Minimal Focus Indicators on PDF 
    - Text size is defined using pixels; UI text doesn't change when user changes browser settings
    - Thank you page: Non-descriptive alt text - View history: Tab order doesn't match focus order

</details>

## Videos
[Download Videos (ZIP)](https://drive.google.com/file/d/143Py73pkHjOUsAzlvQcKuYIfU2fK9Mhd/view?usp=share_link)

## Audit Docs
**Auditors**: @SarahKay8 and @coforma-jamie 
- **Google Drive - Google Sheets**
    - [Form Audit](https://docs.google.com/spreadsheets/d/1elZ23024sbp6Zi0Kpazir7jQRtAqn5NTDgeMY34Q0RM/edit?usp=share_link)
    - [Sign and Submit Audit](https://docs.google.com/spreadsheets/d/1nV6iehJ3kq2FcQkhcC-1yyuwJa1bYM2laQvZjPpcMJg/edit?usp=share_link)
- **Excel Versions** (should access be an issue with Google Drive)
    - [Accessibility audit_ Memorials Self-Service, Apply for a medallion DocuSign form.xlsx](https://github.com/user-attachments/files/16084796/Accessibility.audit_.Memorials.Self-Service.Apply.for.a.medallion.DocuSign.form.xlsx)
    - [Accessibility audit_ Memorials Self-Service, Apply for a medallion (sign and submit) DocuSign form.xlsx](https://github.com/user-attachments/files/16090390/Accessibility.audit_.Memorials.Self-Service.Apply.for.a.medallion.sign.and.submit.DocuSign.form.xlsx)

cc: @coforma-terry @NaomiPMC @shiragoodman
